components/tabs/tab.js

```diff
@@ -0,0 +1,129 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+/**
+ * Tab component
+ */
+class Tab extends Component {
+  handleTabClick = (event) => {
```

Comment:
Could you let me know in which proposal stage this is?

Reply:
@froskie ?

Reply:
![selection_164](https://user-images.githubusercontent.com/10462372/32096631-b00974c8-bb0f-11e7-8b5a-97228ddac643.png)

Reply:
stage 3 as I see https://github.com/tc39/proposal-class-fields